### ISSUE TITLE
Namespace clash with mailchimp package

### ISSUE THREAD

**Issue by eltismerino** [divio/django-mailchimp#63]
This clashes with the [official mailchimp package on pypi](https://pypi.python.org/pypi/mailchimp), which is very unfortunate.

I'd expect the django-mailchimp package to be named something else, even if it's useless to use both packages, one might be forced to have both installed, depending on the project setup.

**Comment by czpython** [comment]
Agree. I've been working on mailchimp 3.0 compatibility in feature/3.0.
This will be a backwards incompatible release, so maybe we can include the renaming in there as well.

**Comment by eltismerino** [comment]
Great. Any ETA on 3.0?
